### PR TITLE
[#5] Release 2.1.0

### PR DESCRIPTION
**Pull request by joebochill**
#### Changes proposed in this Pull Request:
- Includes new translation functions to work with i18N
- Automated publishing scripts
- Updated demo project

### CODE REVIEW

**Review by jeffvg — APPROVED** [review]
looks good

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pxblue/highcharts",
-  "version": "2.0.0",
+  "version": "2.1.0-beta.0",
```
Comment (huayunh) [inline package.json]:
```suggestion
  "version": "2.1.0",
```

**Review by huayunh — CHANGES_REQUESTED** [review]
I assume that this is just to publish the package v2.1.0, and there would be a follow up PR to clean up the demos?

**Review by huayunh — APPROVED** [review]

**Comment by joebochill** [comment]
> I assume that this is just to publish the package v2.1.0, and there would be a follow up PR to clean up the demos?

Yes. The react demo should be pretty clean I think. Angular still needs to be updated. And once this package is published, the demos will be able to use it in their dependencies. Although these changes don’t affect the demo at all, so it can be a lower priority.

**Comment by huayunh** [comment]
I noticed that in the demo, neither Angular nor React is using open sans in Stackblitz / CSB. I am assuming this will get fixed once we a finished with all that theme stuff?

**Pull request merged**